esp/esp/users/models/__init__.py

```diff
@@ -202,12 +202,23 @@ def ajax_autocomplete(cls, data):
 
         query_set = cls.objects.filter(q_names | q_username | q_id)
 
+        if group:
+            query_set = query_set.filter(groups=group)
```

Comment:
Couldn't a user belong to more than one group? (e.g. both "Student" and "Splash Fall 2019 winners") Then will this query do the right thing?

Reply:
Yep, this query still works. My admin/teacher account showed up in the autocomplete for the teacher form.